x-pack/plugins/cases/common/utils/user_actions.ts

```diff
@@ -71,3 +72,7 @@ export const isCreateCaseUserAction = (userAction: unknown): userAction is Creat
 
 export const isUserActionType = (field: string): field is UserActionTypes =>
   ActionTypes[field as UserActionTypes] != null;
+
+export const isCategoryUserAction = (userAction: unknown): userAction is CategoryUserAction =>
```

Comment:
Is this only used in the tests?

Reply:
typescript was complaining in a few places when I didn't have this 😬